app/resources/view.html.template

```diff
@@ -17,15 +17,22 @@
   {{person.full_name}} - {{block.super}}
 {% endblock %}
 
-{% if params.query or params.given_name or params.family_name %}
-  {% block logo %}
+{% block logo %}
+  {% if params.query_name or params.given_name or params.family_name %}
```

Comment:
Do you have any reason to swap the order of "block logo" and this if-statement? The original order looks better because you don't need to put else-statement to repeat the original implementation here.

Or, you can keep the current order, but replace the content of the else-statement with {{block.super}}, which means that it uses the implementation written in the base template.